test/surrogate_escape.html

```diff
@@ -0,0 +1,2 @@
+<a class="dropdown--item_anchor transition--background" href="/learn-spring-security-course" title="The “Learn Spring Security” Course">
+</a>
```

Comment:
AFAICS this doesn't actually trigger the bug you're tring to fix.

Reply:
Thank you jwilk! Looking into it!

Reply:
Dear jwilk! let me know we can produce which characters incase surrogates, Or simply we can ignore them completely and produce nothing